docs/openapi/schemas.yaml

```diff
@@ -919,6 +919,154 @@ SiteTopPageList:
   type: array
   items:
     $ref: './schemas.yaml#/SiteTopPage'
+SiteExperimentVariant:
+  type: object
+  properties:
+    name:
+      description: The name of the variant
+      type: string
+    label:
+      description: The label of the variant
+      type: string
+    split:
+      description: The split allocation of the traffic to the variant
+      type: number
+    url:
+      description: The url of the variant
+      $ref: '#/URL'
+    views:
+      description: The number of views for the variant
+      type: integer
+    interactionsCount:
+      description: The interactions count for the variant
+      type: integer
+    p_value:
+      description: The p-value of the variant for the configured metric or clicks
+      type: number
+    power:
```

Comment:
What does power represent from a statistical point of view?

Reply:
@ramboz 
I'm not really sure how we can use the power to show the information on the UI, but in the Dragos notebooks, I see we're calculating p_value, power and statsig, so I've kept all the values there.
IIRC, high power increases the confidence that a real effect exists.